test/testfunctions.cpp

```diff
@@ -471,10 +471,10 @@ class TestFunctions : public TestFixture {
               "}");
         ASSERT_EQUALS("[test.cpp:7]: (error) Invalid strlen() argument nr 1. A nul-terminated string is required.\n", errout.str());
 
-        check("int f() { char x = '\\0'; return strcmp(""asdf"", &x); }");
+        check("int f() { char x = '\\0'; return strcmp(""Hello world"", &x); }");
```

Comment:
can you please fix this. This code:

    "strcmp(""Hello world"", &x)"

is the same as:

    "strcmp("  "Hello world"  ", &x)"

which just means that you concatenate the strings. So the test case becomes:

    strcmp(Hello world, &x)

Please escape the code:

    strcmp(\"Hello world\", &x);

Reply:
I think that `asdf` is fine. Feel free to replace `Hello world` with `asdf`.

Reply:
Oops, fixed it. I didn't bother changing Hello world.